### ISSUE TITLE
Error with view rendering

### ISSUE THREAD

**Issue by debeniter86** [barryvdh/laravel-form-bridge#12]
Hi,
I got error while running following code. Please help me.
### Controller -
-----------
```
use Symfony\Component\Form\Extension\Core\Type\FormType;
use Symfony\Component\Form\Extension\Core\Type\TextType;
use Symfony\Component\Form\Extension\Core\Type\EmailType;
use Symfony\Component\Form\Extension\Core\Type\SubmitType;
public function create(\Illuminate\Http\Request $request)
    {
        $user = \App\User::first();
        $form = app('form.factory')->create(FormType::class, $user)
            ->add('name', TextType::class)
            ->add('email', EmailType::class)
            ->add('save', SubmitType::class, array('label' => 'Save user'));
        $form->handleRequest($request);
        return view('sessions.create',['form' => $form->createView()]);
    }
```
### View 
--------
 ```
  {{ form_start(form) }}
  {{ form_widget(form) }}
  {{ form_end(form) }}
```
### Error-
----------
```
 ErrorException in create.twig line 15:
The "Symfony\Bridge\Twig\Extension\FormExtension" extension is not enabled 
```

**Comment by SwenVanZanten** [comment]
Same here, this happends from twig 1.26.0 and up..
But I think this problem belongs to the twig package: https://github.com/rcrowe/TwigBridge

**Comment by barryvdh** [comment]
Yes you can use twig 1.24, not sure why it happens/.

**Comment by debeniter86** [comment]
This is my require composer.json file. Where I will change twig 1.24 ?

` "require": {
        "php": ">=5.5.9",
        "laravel/framework": "5.1.*",
        "cartalyst/sentinel": "2.0.*",
        "laravelcollective/html": "5.1.*",
        "guzzlehttp/guzzle": "~5.3|~6.0",
        "ircmaxell/password-compat": "^1.0",
        "rcrowe/twigbridge": "^0.9.3",
        "barryvdh/laravel-form-bridge": "^0.2.1"
    },`

**Comment by SwenVanZanten** [comment]
Just require `"twig/twig": "1.25.*"` yourself 👍 
This will overwrite the twig requirement by the `rcrowe/twigbridge` package.

**Comment by debeniter86** [comment]
Thanks @HDsign .

**Comment by barryvdh** [comment]
It's possible that it now works again with the latest twig, but haven't confirmed that yet.

**Comment by barryvdh** [comment]
Should be fixed with v0.2.2 now :)
